sds_data_manager/lambda_code/SDSCode/batch_starter.py

```diff
@@ -270,25 +255,23 @@ def prepare_data(
     #     "--instrument", "mag",
     #     "--data-level", "l1a",
     #     "--start-date", "20231212",
-    #     "--end-date", "20231212",
-    #     "--version", "v00-01",
+    #     "--version", "v001",
     #     "--dependency", """[
     #         {
     #             'instrument': 'swe',
     #             'data_level': 'l0',
     #             'descriptor': 'lveng-hk',
     #             'start_date': '20231212',
-    #             'end_date': '20231212',
-    #             'version': 'v01-00',
+    #             'version': 'v001',
     #         },
     #         {
     #             'instrument': 'mag',
     #             'data_level': 'l0',
     #             'descriptor': 'lveng-hk',
     #             'start_date': '20231212',
-    #             'end_date': '20231212',
-    #             'version': 'v00-01',
+    #             'version': 'v001',
     #         }]""",
+    #    "--repointing", "None | int",
```

Comment:
The other ones are explicit examples, so maybe make this one explicit too? I'm not sure `--repointing None` would actually work...
```suggestion
    #    "--repointing", 1,
```